web-src/src/components/NavbarBottom.vue

```diff
@@ -106,8 +106,9 @@
         </div>
       </div>
     </div>
-    <div class="navbar-brand is-flex-grow-1">
-      <navbar-item-link :to="{ name: 'queue' }" exact class="mr-auto">
+   <div class="navbar-brand is-flex-grow-1">
+      <group class="navbar-item is-expanded is-justify-content-left is-no-basis">
```

Comment:
Where does this group element come from?
It doesn't appear to be an HTML element.
Could you please explain?
Thanks

Reply:
The bottom bar is functionally 3 groups of items. There's one (group of) item on the left for switching between queue mode and "now playing mode", the center group of playing buttons and the right group for toggling lyrics and showing the output panes.

Initially, there was an hack by using bulma's navbar (that was not made for splitting the bar in 3 groups) done by using right margin auto on the left item, and left margin on the right item. This worked because the number of elements was odd and all the same size (except for the center).

The change I've made is to switch to bulma's flex classes, when each group of items must be a direct children of the navbar-brand. By using flex, the centering of items in the box is now clearly defined (justify left, center, and right) and doesn't depend on the item size and count. 

I've had to add `is-no-basis` since it's bulma doesn't have a class for `flex-basis:0`. 

I've used the element `group` because it's a group. You are allowed to use any tag name in HTML5, and I'm not using any CSS rule with that tag name. It's possible to use a `div` if you prefer.

Reply:
Thanks for the explanation.
I thought the group was some sort of special element I didn't know.
For clarity, and because it's not a specific component like the element `<mdicon>`, I would recommend using a simple `div` in this situation.

Reply:
Done